client/actions/stats/getCategoryCompanyGender.js

```diff
@@ -0,0 +1,24 @@
+import request from 'superagent'
+import {showError} from '../index'
+
+export const GET_COMPANY_GENDER = 'GET_COMPANY_GENDER'
```

Comment:
rename to `RECEIVE_COMPANY_GENDER_STATS` to be consistent with other action types

Reply:
done